### PR TITLE
Updates setup_perf_cluster.sh to support uninstall.

### PR DESCRIPTION
**Pull request by danehans**
This PR adds support for uninstalling what was done by `setup_all` in `setup_perf_cluster.sh` script.

Fixes https://github.com/istio/istio/issues/3023

### CODE REVIEW

File: tools/setup_perf_cluster.sh [hunk 1]
```diff
@@ -77,6 +81,13 @@ function create_vm() {
   sleep 45
 }
 
+function delete_vm() {
+  echo "Deleting VM_NAME=$VM_NAME"
+  Execute gcloud compute instances delete $VM_NAME $GCP_OPTS -q
+  echo "Waiting a bit for the VM to be deleted..."
+  sleep 45
```
Comment (ldemailly) [inline tools/setup_perf_cluster.sh]: why wait ?

File: tools/setup_perf_cluster.sh [hunk 2]
```diff
@@ -238,6 +260,13 @@ function setup_all() {
   setup_cluster_all
 }
 
+function delete_all() {
+  delete_istio
```
Comment (ldemailly) [inline tools/setup_perf_cluster.sh]: let’s add a confirmation message and sleep for a chance for the user to control C out first

**Comment by andraxylia** [comment]
/lgtm
comments can be addressed in another PR

**Comment by ldemailly** [comment]
this was not to be merged - please don't merge PR that don't have accept

**Comment by ldemailly** [comment]
@danehans apologies for the churn/noise but let's complete this properly before we merge it

**Comment by danehans** [comment]
@ldemailly since the PR has merged, is submitting a follow-on PR to address your comments the best course of action?

**Comment by ldemailly** [comment]
it's not really merged, it's merged then revert - apologies for the churn

can you either reopen this one (maybe adding a commit will reopen it) or make a new one ?

**Pull request merged**